todo_app/templates/index.html

```diff
@@ -9,10 +9,21 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2>Task Master</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in get_items() %}
+        <li>{{item.title}}</li>
+         {% endfor %}
+          </div>
+        
       </ul>
+      <!-- setting the URL path for where the form is to be 'posted' to -->
+      <form action= "/newitem" method= "post">
+        
+        <input type="text" name="title">
+        <input type="submit" value="submit"> 
```

Comment:
Minor - might be nice to have a `required` attribute on this input to prevent people from adding empty tasks!